docs/release-notes/1289-streaming-ednpoints-coroutine-leak

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**BUG FIXES**
+
+- The log stream handlers could get into a situation where the requester
+has abandoned the request yet the handler still stays up and polls for
+new entries indefinitely. This fixes that by checking the context.
```

Comment:
This is way more detailed than what the users who might read this would understand.  I'd say something like "Fixed a performance issue during streaming trial logs.", or not mention it at all (only if we know that no users have encountered it).

Reply:
I dunno, gory details like this seem fine to me. I agree it is more detail than we strictly _need_ and I wouldn't require it for every change, but it does provide useful context that is missing from a vague statement like "Fixed a performance issue during streaming trial logs." -- e.g., the nature of the performance problem, the situation in which it occurs, etc.

Reply:
so stick with the current one?

Reply:
:rocket: